scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -118,63 +124,98 @@ workflow GvsCreateFilterSet {
       preemptible_tries = 3,
   }
 
-  call Tasks.IndelsVariantRecalibrator {
-    input:
-      sites_only_variant_filtered_vcf = MergeVCFs.output_vcf,
-      sites_only_variant_filtered_vcf_index = MergeVCFs.output_vcf_index,
-      recalibration_filename = filter_set_name + ".indels.recal",
-      tranches_filename = filter_set_name + ".indels.tranches",
-      recalibration_tranche_values = ["100.0", "99.95", "99.9", "99.5", "99.0", "97.0", "96.0", "95.0", "94.0", "93.5", "93.0", "92.0", "91.0", "90.0"],
-      recalibration_annotation_values = indel_recalibration_annotation_values,
-      mills_resource_vcf = mills_resource_vcf,
-      mills_resource_vcf_index = mills_resource_vcf_index,
-      axiomPoly_resource_vcf = axiomPoly_resource_vcf,
-      axiomPoly_resource_vcf_index = axiomPoly_resource_vcf_index,
-      dbsnp_resource_vcf = dbsnp_vcf,
-      dbsnp_resource_vcf_index = dbsnp_vcf_index,
-      use_allele_specific_annotations = true,
-      disk_size = "1000",
-      machine_mem_gb = INDEL_VQSR_mem_gb_override,
-      max_gaussians = INDEL_VQSR_max_gaussians_override,
-      maximum_training_variants = INDEL_VQSR_maximum_training_variants,
+  # From this point, the paths diverge depending on whether they're using classic VQSR or VQSR-Lite
+  # The first branch here is VQSR-Lite, and the second is classic VQSR
+  if (!use_classic_VQSR) {
+    call VQSRLite.JointVcfFiltering as JointVcfFiltering {
+      input:
+        vcf = ExtractFilterTask.output_vcf,
+        vcf_index = ExtractFilterTask.output_vcf_index,
+        sites_only_vcf = MergeVCFs.output_vcf,
+        sites_only_vcf_index = MergeVCFs.output_vcf_index,
+        basename = filter_set_name,
+        gatk_docker = "us.gcr.io/broad-gatk/gatk:4.3.0.0",
```

Comment:
Good question.  It looks as though that was necessary when Bec yanked the code from master (which is at GATK 4.3.0) into our branch.  It may have been dependent on the latest version of GATK?  Maybe we leave it as an input with that as the default value and see if we can remove it if/when our branch catches up to master.